src/main/java/kernbeisser/Tasks/Users.java

```diff
@@ -125,7 +128,7 @@ private static void generateUsername(HashSet<String> usernames, User user) {
     }
   }
 
-  public static void switchUserGroup(int userId, int userGroupId) {
+  public static boolean switchUserGroup(int userId, int userGroupId) {
```

Comment:
returning a boolean instead of throwing the exception? why?
This is clearly a mix with model and view I guess

Reply:
Ok bad style, sorry! Better throw, and than catch the exception in the controller. What is required though is user feedback on missing permissions